### DISCUSSION TITLE
Dual CPU Windows Plot Times

### DISCUSSION THREAD

**Discussion by mrkkwar1** [Chia-Network/chia-blockchain#416] [Q&A] [answered]
Hello guys, 

I have a Dual CPU Server and I would think there would be better plot times then my laptop any ideas?

Laptop
I7-9570H
10GB RAM given to plot 
PC601 NVMe SK hynix 1TB
k=31 7.85Hours 

2 ×Intel Xeon E5-2670 v2 @ 2.50GHz
10GB RAM given to plot 
Sabrent 2TB Rocket NVMe PCIe M.2 2280
k=31 9.93 hours

Attached plot log Laptop
[plotter_log.txt](https://github.com/Chia-Network/chia-blockchain/files/5250526/plotter_log.txt)

Attached plot log SERVER
[plotter_log.txt](https://github.com/Chia-Network/chia-blockchain/files/5250516/plotter_log.txt)

**Comment by hoffmang9** [comment, marked as answer]
The current plotter is single threaded. You can either run multiple instances on the command line or wait for Beta 14 which will likely have multithreading.

**Reply by mrkkwar1** [reply]
Thanks!
